test/e2e/toolchain_test.go

```diff
@@ -40,10 +41,15 @@ func TestToolchain(t *testing.T) {
 	cheOg := cheinstallation.NewOperatorGroup(cheOperatorNS)
 	cheSub := cheinstallation.NewSubscription(cheOperatorNS)
 	cheCluster := cheinstallation.NewCheCluster(cheOperatorNS)
+	installation := tektoninstallation.NewInstallation()
```

Comment:
I know that the name `tektoninstallation` is already taken by the package name, but could you please rename the variable at least to `tknInstallation` to be clear what the variable stands for...?